lib/spack/spack/cmd/develop.py

```diff
@@ -38,28 +40,22 @@ def setup_parser(subparser):
         "--clone",
         action="store_true",
         dest="clone",
-        default=None,
+        default=True,
```

Comment:
I don't understand this change -- this seems to suggest in the default case we clone regardless of whether the path is already cloned?

Reply:
Need to change the help messages.

Reply:
@becker33 updated this one